### PR TITLE
Adds basic scaffold for components/system

### PR DESCRIPTION
**Pull request by rafaeldelboni**

### CODE REVIEW

File: src/org/parenthesin/components/webserver.clj [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  (:require [com.stuartsierra.component :as component]))
+
+(defrecord WebServer [config routes storage]
+  component/Lifecycle
+  (start [this]
+    (println ";; Starting webserver")
+    (println "port:" (get-in config [:config :webserver-port]))
+    (println "cors:" (get-in config [:config :cors-allowed-origins]))
+    (println "unsplash:" (get-in config [:config :unsplash-access-key]))
+    (assoc this :service {}))
```
Comment (lucasdf) [inline src/org/parenthesin/components/webserver.clj:11]: shouldn't we do something here, or this will be implemented later?

Comment (rafaeldelboni) [reply src/org/parenthesin/components/webserver.clj:11]: I was planning to push the components part on this PR and later adds the webserver functions.

Then later when I need a scaffold for an empty component system I can come back to this PR 
hahaha

Comment (lucasdf) [reply src/org/parenthesin/components/webserver.clj:11]: ~nevermind, I get it :)~ great! :)

**Review by lucasdf — APPROVED** [review]
:clap:

**Pull request merged**